src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -724,7 +724,11 @@ private function resolveServices($value, $file, $isParameter = false)
                     throw new InvalidArgumentException(sprintf('"!service_locator" tag only accepts maps of "@service" references in "%s".', $file));
                 }
             }
-            if (\in_array($value->getTag(), ['tagged', 'tagged_locator'], true)) {
+            if (\in_array($value->getTag(), ['tagged', 'tagged_iterator', 'tagged_locator'], true)) {
+                if ('tagged' === $value->getTag()) {
+                    @trigger_error('"!tagged" tag is deprecated and will be removed in Symfony 5. Please use "!tagged_iterator"', E_USER_DEPRECATED);
```

Comment:
all deprecation template should follow a similar template:
foo is deprecated since Symfony X.Y, use [...] instead.

Reply:
Done